### ISSUE TITLE
Make location search text box accessible and add label and button

### ISSUE THREAD

**Issue by jamestyack** [UnlockedLab/unlockphilly#48]
This box is really great but Liz suggested it should have go button and also there is no label. Please make it clear that box.allows place name or location.
